include/aie/Dialect/AIE/Transforms/AIEPathFinder.h

```diff
@@ -140,21 +141,21 @@ class Pathfinder {
 } // namespace xilinx::AIE
 
 namespace llvm {
```

Comment:
It's weird to add this to llvm::?

Reply:
What's weird? the `DenseMap` specialization or the `using`?